barretenberg/cpp/src/barretenberg/ultra_honk/oink_prover.cpp

```diff
@@ -94,9 +101,12 @@ template <IsUltraFlavor Flavor> void OinkProver<Flavor>::execute_wire_commitment
     // We only commit to the fourth wire polynomial after adding memory recordss
     {
         BB_OP_COUNT_TIME_NAME("COMMIT::wires");
-        witness_commitments.w_l = commitment_key->commit(proving_key->proving_key.polynomials.w_l);
-        witness_commitments.w_r = commitment_key->commit(proving_key->proving_key.polynomials.w_r);
-        witness_commitments.w_o = commitment_key->commit(proving_key->proving_key.polynomials.w_o);
+        witness_commitments.w_l =
```

Comment:
I guess you could do `const auto& commitment_key = proving_key->proving_key.commitment_key;` inside this function, before committing to all the wires, for more succintness.